model/src/test/test_latent_admissions.py

```diff
@@ -60,16 +60,16 @@ def test_admissions_sample():
     )
 
     hosp1 = HospitalAdmissions(
-        infection_to_admission_interval=inf_hosp,
-        infect_hosp_rate_dist=DistributionalRV(
+        infection_to_admission_interval_rv=inf_hosp,
+        infect_hosp_rate_rv=DistributionalRV(
```

Comment:
One could argue for `distrv` here. Depends on what you value. 

Pros:

* More information to the reader

Cons:

* Longer than `rv`
* Now need to do this for every non-just-`rv`
* Using `rv` for all types of `rv`, regarding of `rv`-type, is parsimonious

Reply:
It could be a deterministic random variable, so I think the current name is good.